helmchart/sdp/values.yaml

```diff
@@ -111,7 +111,7 @@ sdp:
   image:
     repository: stellar/stellar-disbursement-platform-backend
     pullPolicy: Always
-    tag: "latest"
+    tag: "3.0.0-rc.1"
```

Comment:
We started fixing the version of the images in the helm chart here https://github.com/stellar/helm-charts/blob/main/charts/stellar-disbursement-platform/values.yaml#L114 

I assume we should do it in this repo as well ?